LEAF_Request_Portal/sources/Form.php

```diff
@@ -2842,36 +2842,433 @@ public function changeFormType($recordID, $categories)
         return 1;
     }
 
+ /**
+     * @param array $query - this is the decoded string used for query
+     * @return bool
+     */
+    private function isProcessQuery(array $query): bool
+    {
+        
+        $externalProcessQuery = false;
+        if (isset($query['limit']) && is_numeric($query['limit'])) {
+
```

Comment:
The change in print_form.tpl identifies another condition that we can account for, and avoid having to add the limit over there:  if there's a recordID term, it wouldn't be considered a large query since there'd only be at most one record.